packages/icons/web/v2/plus_tier_16.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="36" height="16" fill="none" viewBox="0 0 36 16">
+    <g clip-path="url(#plus_tier_16_svg__clip0_11228_178)">
```

Comment:
The referenced clip-path element doesn't exist, which makes me think it's not needed.